package.json

```diff
@@ -1,24 +1,23 @@
 {
   "name": "atd-product",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
I think we should take this project `1.0.0`. It's been doing its job admirably for no less than five years, based on the oldest PR against the repo. Whatcha'll think?